src/AWS.Deploy.Orchestration/OptionSettingHandler.cs

```diff
@@ -210,8 +210,8 @@ public OptionSettingItem GetOptionSetting(RecipeDefinition recipe, string? jsonP
                 optionSetting = optionSettings.FirstOrDefault(os => os.Id.Equals(ids[i]));
                 if (optionSetting == null)
                 {
-                    throw new OptionSettingItemDoesNotExistException(DeployToolErrorCode.OptionSettingItemDoesNotExistInRecipe, $"An option setting item with the specified fully qualified Id '{jsonPath}' cannot be found in the" +
-                    $" '{recipe.Name}' recipe.");
+                    throw new OptionSettingItemDoesNotExistException(DeployToolErrorCode.OptionSettingItemDoesNotExistInRecipe, $"An option setting item with the Id '{ids[i]}' " +
```

Comment:
I had a typo in my recipe which tripped my up for a bit, this would have helped. It's otherwise unrelated to the specific change.